pack/tcu/wos_encounter.json

```diff
@@ -0,0 +1,811 @@
+	},
+	{
+		"clues": 0,
+		"code": "05172",
+		"back_link": "05172b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 12,
+		"faction_code": "mythos",
+		"flavor": "Only witches and criminals occupy these graves. Their tombstones have been left unmarked so as to forever erase the memory of their foul deeds.",
+		"illustrator": "Emilio Rodriguez",
+		"name": "Heretics' Graves",
+		"pack_code": "wos",
+		"position": 172,
+		"quantity": 1,
+		"shroud": 4,
+		"text": "<b>Forced</b> - After a [[Witch]] enemy at Heretics' Graves is defeated: Discard the top 2 cards of the standard encounter deck.",
+		"type_code": "location"
+	},
+	{
+		"clues": null,
+		"code": "05173b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 13,
+		"faction_code": "mythos",
+		"illustrator": "Jose Vega",
+		"hidden": true,
+		"name": "Chapel Crypt",
+		"pack_code": "wos",
+		"position": 173,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "While there are no ready enemies at Chapel Crypt, it gets -3 shroud.\n<b>Haunted</b> - Spawn the top card of your deck facedown, engaged with you. Treat that card as a Reanimated Dead enemy with 1 fight, 1 health, 1 evade, 1 damage, and the [[Monster]] trait.",
+		"traits": "Spectral.",
+		"type_code": "location"
+	},
+	{
+		"clues": 0,
+		"code": "05173",
+		"back_link": "05173b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 13,
+		"faction_code": "mythos",
+		"flavor": "A narrow stone staircase guides you deep below the chapel, where those of good repute were given “proper” burials. In a far corner of the room, a strange, dusty contraption is connected to one of the coffins, as dead as everything else in this room.",
+		"illustrator": "Jose Vega",
+		"name": "Chapel Crypt",
+		"pack_code": "wos",
+		"position": 173,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "While there are no ready enemies at Chapel Crypt, it gets -3 shroud.",
+		"type_code": "location"
+	},
+	{
+		"clues": null,
+		"code": "05174b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 14,
+		"faction_code": "mythos",
+		"illustrator": "Jose Vega",
+		"hidden": true,
+		"name": "Chapel Crypt",
+		"pack_code": "wos",
+		"position": 174,
+		"quantity": 1,
+		"shroud": 6,
+		"text": "While there are no [[Hex]] treacheries among investigators' threat areas at Chapel Crypt, it gets -3 shroud.\n<b>Haunted</b> - Find the topmost [[Hex]] treachery in the standard encounter discard pile and put it into play in your threat area.",
+		"traits": "Spectral.",
+		"type_code": "location"
+	},
+	{
+		"clues": 0,
+		"code": "05174",
+		"back_link": "05174b",
+		"encounter_code": "the_wages_of_sin",
+		"encounter_position": 14,
+		"faction_code": "mythos",
+		"flavor": "A narrow stone staircase guides you deep below the chapel, where those of good repute were given “proper” burials. In a far corner of the room, a strange, dusty contraption is connected to one of the coffins, as dead as everything else in this room.",
```

Comment:
please change " around 'proper' to \": 'where those of good repute were given \"proper\" burials'